presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidIntegrationSmokeTest.java

```diff
@@ -272,4 +287,73 @@ public void testFilteringForTablesAndColumns()
+        assertAggregationPushedDown("SELECT stddev(double_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT stddev_samp(double_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT stddev_pop(double_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT variance(double_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT var_samp(double_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT var_pop(double_col) FROM singlerow");
+
+        // for bigint
+        assertAggregationPushedDown("SELECT count(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT min(shippriority) FROM orders group by custkey");
+        assertAggregationPushedDown("SELECT max(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT sum(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT avg(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT stddev(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT stddev_samp(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT stddev_pop(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT variance(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT var_samp(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT var_pop(shippriority) FROM orders");
+        assertAggregationPushedDown("SELECT stddev(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT stddev_samp(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT stddev_pop(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT variance(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT var_samp(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT var_pop(bigint_col) FROM nodata");
+        assertAggregationPushedDown("SELECT stddev(bigint_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT stddev_samp(bigint_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT stddev_pop(bigint_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT variance(bigint_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT var_samp(bigint_col) FROM singlerow");
+        assertAggregationPushedDown("SELECT var_pop(bigint_col) FROM singlerow");
+
+        assertAggregationPushedDown("SELECT approx_distinct(custkey) FROM orders", ImmutableList.of(Optional.of(100L)));
+        assertAggregationPushedDown("SELECT approx_distinct(totalprice) FROM orders", ImmutableList.of(Optional.of(100L)));
+        assertAggregationPushedDown("SELECT approx_distinct(comment) FROM orders", ImmutableList.of(Optional.of(100L)));
+        assertAggregationPushedDown("SELECT approx_distinct(__time) FROM orders", ImmutableList.of(Optional.of(100L)));
+    }
```

Comment:
Please add some tests:

- with GROUP BY one column
- with GROUP BY two columns
- with GROUP BY GROUPING SETS
- with SELECT DISTINCT
- with WHERE

Reply:
modified some of the sqls so it has those conditions. I didn't add one for grouping set as it is not pushed down yet.